rust/remotivelabs-broker/src/remotive_api/base.rs

```diff
@@ -0,0 +1,1260 @@
+//// Symbolic message representing no argument needed or provided.
```

Comment:
Should we maybe remove this file and add it to .gitignore?

Reply:
Removed file